MagneticField/VolumeBasedEngine/src/MagGeometry.cc

```diff
@@ -110,7 +134,11 @@ MagVolume const* MagGeometry::findVolume1(const GlobalPoint& gp, double toleranc
   MagVolume6Faces const* found = nullptr;
 
   int errCnt = 0;
-  if (inBarrel(gp)) {  // Barrel
+
+  float R = gp.perp();
+  float Z = fabs(gp.z());
```

Comment:
In this case I think it's clear what the variables mean, as they're just the coordinate name, but I can rename to `r` and `z`, or `rho` and `z` since we are talking about cylindrical coordinates.